components/layout_2020/taffy/layout.rs

```diff
@@ -0,0 +1,600 @@
+
+                        // Adjust known_dimensions from border box to content box
+                        let pbm = non_replaced.style.padding_border_margin(containing_block);
+                        let margin_sum = pbm.margin.auto_is(Au::zero).sum();
+                        let content_box_inset =
+                            (pbm.padding_border_sums + margin_sum).map(|v| v.to_f32_px());
+                        let content_box_known_dimensions =
+                            taffy::Size {
+                                width: inputs.known_dimensions.width.map(|width| {
+                                    width - pbm.padding_border_sums.inline.to_f32_px()
+                                }),
+                                height: inputs.known_dimensions.height.map(|height| {
+                                    height - pbm.padding_border_sums.block.to_f32_px()
+                                }),
+                            };
+
+                        // Compute inline size
+                        let inline_size = content_box_known_dimensions.width.unwrap_or_else(|| {
+                            let inline_sizes =
+                                non_replaced.inline_content_sizes(&self.layout_context);
+                            let adjusted_available_space = inputs
+                                .available_space
+                                .width
+                                .map_definite_value(|width| width - content_box_inset.inline);
+
+                            resolve_content_size(adjusted_available_space, inline_sizes)
+                        });
+
+                        let maybe_block_size =
+                            option_f32_to_lpa(content_box_known_dimensions.height);
+                        let content_box_size_override = ContainingBlock {
+                            inline_size: Au::from_f32_px(inline_size),
+                            block_size: maybe_block_size,
+                            style: &non_replaced.style,
+                        };
+
+                        let layout = {
+                            let mut child_positioning_context = PositioningContext::new_for_subtree(
+                                self.positioning_context
+                                    .collects_for_nearest_positioned_ancestor(),
+                            );
+                            let layout = non_replaced.layout(
+                                &self.layout_context,
+                                &mut child_positioning_context,
+                                &content_box_size_override,
+                                &containing_block,
+                            );
+
+                            // Store layout data on child for later access
+                            child.positioning_context = child_positioning_context;
+
+                            // self.positioning_context.append(child_positioning_context);
```

Comment:
Looks like this should be removed as well.

Reply:
Still not removed